Lib/test/test_os.py

```diff
@@ -3089,6 +3089,14 @@ def kill_process(pid):
 
 @support.requires_subprocess()
 class SpawnTests(unittest.TestCase):
+    @staticmethod
+    def quote_args(args):
+        # On Windows, os.spawn* simply joins arguments with spaces:
+        # arguments need to be quoted
+        if not os.name == 'nt':
+            return args
+        return [f'"{arg}"' if " " in arg.strip() else arg for arg in args]
```

Comment:
What if there is a double quote in the path?

Reply:
That's not possible on Windows, as the double quote character [is not allowed in Windows file paths](https://en.wikipedia.org/wiki/Filename#In_Windows), so this quoting strategy is sufficient for its purpose here. If one did need to handle arbitrary arguments, the undocumented [`subprocess.list2cmdline`](https://github.com/python/cpython/blob/bef9efabc3f899a8f05cc6bee009c7c5fb3d01ae/Lib/subprocess.py#L576) could be used instead.